ui/selectors/transactions.js

```diff
@@ -10,8 +10,11 @@ import {
   TRANSACTION_TYPES,
 } from '../../shared/constants/transaction';
 import { transactionMatchesNetwork } from '../../shared/modules/transaction.utils';
-import { getCurrentChainId, deprecatedGetCurrentNetworkId } from './selectors';
```

Comment:
The main change here is to have `getSelectedAddress` come from `ui/selectors/selectors` and not from `ui/selectors`. The previous change created a circular dependency between `ui/selectors` and `ui/selectors/transactions`, which prevents us from `jest.mock`'ing `ui/selectors` correctly in component tests.